packages/twenty-front/src/modules/workflow/components/WorkflowEditActionFormServerlessFunction.tsx

```diff
@@ -23,6 +28,53 @@ export const WorkflowEditActionFormServerlessFunction = (
 
   const { serverlessFunctions } = useGetManyServerlessFunctions();
 
+  const form = useForm({
+    disabled: props.readonly,
+  });
```

Comment:
I think we don't want to use useForm anymore, can a useState be enough?

Reply:
I know the team wants to ditch react-hook-form, but it will make it easy to add validation to the inputs. We can do it without react-hook-form, but I don't know if we already have primitives to manage forms.

Reply:
We probably need to learn to use it properly indeed